### PR TITLE
NYPL and TIB Sponsors

### PR DESCRIPTION
**Pull request by JeromeMartinez**
Thanks  <a href="https://www.tib.eu">Leibniz Information Centre for Science and Technology (TIB)</a>!

### CODE REVIEW

File: src/QCToolsBundle/Resources/views/Default/index.html.twig [hunk 1]
```diff
@@ -39,8 +39,12 @@
...
-                MediaArea was involved in the initial development and provides binaries for all platforms.</p>
+                Initially Funded by the <a href="https://www.neh.gov/" target="_blank">National Endowment for the Humanities</a> and the <a href="http://www.knightfoundation.org" target="_blank">Knight Foundation</a>; designed and led by Dave Rice and the <a href="https://www.bavc.org/" target="_blank">Bay Area Video Coalition</a>; developed by MediaArea, <a href="https://github.com/utzig">Fabio Utzig</a>, <a href="https://github.com/ElderOrb">Alexander Ivash</a>.<br>
+                Version 1.2 with support of panel tracks and QCVault is brought to you by <a href="https://www.tib.eu">Leibniz Information Centre for Science and Technology (TIB)</a>.<br>
```
Comment (retokromer) [inline src/QCToolsBundle/Resources/views/Default/index.html.twig:44]: You could use here as well the shorter `https://tib.eu`.

Comment (JeromeMartinez) [reply src/QCToolsBundle/Resources/views/Default/index.html.twig:44]: their official URL is with `www`, the URL without `www` redirects to the one with it so slower for the person who clicks on the URL :p.

Comment (retokromer) [reply src/QCToolsBundle/Resources/views/Default/index.html.twig:44]: Below you use the short form…

Comment (JeromeMartinez) [reply src/QCToolsBundle/Resources/views/Default/index.html.twig:44]: > Below you use the short form…

Incoherent, true. Fixed.

**Pull request merged**